docs/ExportToCsv.md

```diff
@@ -0,0 +1,36 @@
+# Export To Csv
+
+The ToCsv operation allows a user to export data from a running Gaffer instance to an iterable of CSV formatted strings.
+This operation can be chained together with ExportToLocalFile, which will take the iterable of strings and write them to a 
+local file. The ToCsv operation requires a CsvGenerator, this can be built by the user, alternatively it can be autogenerated by 
+supplying the operation with a CsvFormat. CsvFormats that are currently supported include those used by Neo4j and AWS's 
+Neptune graphs, both of which are based on the openCypher [format](./openCypher) but have slight differences.
```

Comment:
There doesn't seem to be other documentation for what CsvFormats are. Maybe there could be some more mention of them or the classes used?

Reply:
Could even make sub pages for each format and describe what each header looks like, post an example file, and link to the appropriate reference page (neo4j or Neptune)